README.md

```diff
@@ -2,18 +2,28 @@
 
 Bodywork uses Kubernetes for running machine learning workloads and services, because we believe that Kubernetes comes shipped with all the resources required for building an effective Machine Learning Operations ([MLOps](https://en.wikipedia.org/wiki/MLOps)) platform.
 
 ## What Problems Does Bodywork Solve?
 
-Containerising machine learning code using Docker, pushing the build artefacts to an image repository and then configuring Kubernetes to orchestrate these into batch jobs and services, requires skills and expertise that most machine learning engineers do not have the time (and often the desire) to learn.
+Running machine learning code in containers has become a common pattern to guarantee reproducibility between what has been developed and what is deployed in production.
 
-This is where Bodywork steps-in - to make sure that your code is delivered to the right place, at the right time, so that your models are trained, deployed and available to the rest of your team. Bodywork will:
+The process of containerising machine learning code using Docker to build images, pushing the build artefacts to an image repository and then configuring a container orchestration platform to run batch workloads and deploy services, requires skills and expertise that most machine learning engineers do not have the time (and often the desire) to learn.
 
-- automate the configuration of Kubernetes jobs and deployments to run complex machine learning workflows that result in machine learning service deployments.
-- continuously deliver machine learning code - for training models and defining model-scoring services - directly from your Git repository into running containers on Kubernetes.
+Scale this scenario to one where there are multiple models to worry about, all needing to be re-trained and re-deployed, and it is easy to see how quickly the management of these pipelines will become a large and undesirable burden.
+
+This is where the Bodywork MLOps framework steps-in - to take care of delivering your code to the right place and executing it at the right time, so that your models are always trained, deployed and available to the rest of your product team.
+
+![bodywork_diagram](https://bodywork-media.s3.eu-west-2.amazonaws.com/visual_explanation.png)
+
+Bodywork is a tool built upon the Kubernetes container orchestration platform and is aimed at machine learning engineers to help them:
+
+- **continuously deliver code** - for training models and defining model-scoring services, by having running containers on Kubernetes pull it directly from your project's Git repository, and so removing the need to build-and-push container images; and,
```

Comment:
"by having running" - Use either having or running, not both.  Or maybe "using"?

"and so" - Therefore or thereby would sound better. 

Remove trailing "and,"

Reply:
- **Continuously deliver code** - for training models and defining model-scoring services. Bodywork containers running on Kubernetes will pull code directly from your project's Git repository, removing the need to build-and-push your own container images.

Reply:
This is much better - thanks.